Vault-Deployment/variables.tf

```diff
@@ -56,3 +56,13 @@ variable "additionalDomains" {
   type    = list(string)
   default = []
 }
+
+variable "enable_kubernetes_network_policies" {
```

Comment:
@norbertgruszka I added two input variables where you can deploy Kubernetes and/or Cilium policies